packages/table-next/src/react/index.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  SortAscIcon,
+  SortDescIcon,
+  SortIcon
+} from '@pluralsight/ps-design-system-icon'
+import {
+  names as themeNames,
+  useTheme
+} from '@pluralsight/ps-design-system-theme'
+import { HTMLPropsFor, ValueOf } from '@pluralsight/ps-design-system-util'
+import { compose, css } from 'glamor'
+import React from 'react'
+
+import stylesheet from '../css'
+import { aligns, sorts } from '../vars'
+
+const styles = {
+  container: (
+    themeName: ValueOf<typeof themeNames>,
+    opts: { scrollable?: boolean }
```

Comment:
Looks like object not needed. Just pass bool?

Reply:
no reason. just habit of passing objects to the style functions like we've done in other components